torch/_export/converter.py

```diff
@@ -1015,6 +1013,17 @@ def __init__(
         self.lift_tensor_constants_to_buffer()
 
     def convert(self) -> ExportedProgram:
+        log.info("""
+TS2EPConverter logging starts from here.
+
+INFO: (TORCH_LOGS="export" <cmd>)
+    * Log TorchScript IR.
+
+DEBUG: (TORCH_LOGS="+export" <cmd>), additionally
+    * Log conversion IR by IR in a format of [<conversion handler name>] converts [<IR>].
+        """)
+        log.info(f"TorchScript graph\n\n{self.ts_graph}\n")  # noqa: G004
```

Comment:
This seems expensive to log:
```suggestion
        log.info(f"TorchScript graph\n\n%s\n", {self.ts_graph})
```